bluemira/geometry/tools.py

```diff
@@ -813,9 +813,7 @@ def split_wire(wire: BluemiraWire, vertex: Iterable, tolerance: float = EPS):
     GeometryError:
         If the vertex is further away to the wire than the specified tolerance
     """
-    wire_1, wire_2 = cadapi.split_wire(
-        wire.get_single_wire()._shape, vertex, tolerance=tolerance
-    )
+    wire_1, wire_2 = cadapi.split_wire(wire.shape, vertex, tolerance=tolerance)
```

Comment:
Here you change to `.shape` but in many of the other functions in tools.py, they still use `._shape`. Is there a reason it is different here?